central/sensor/service/connection/connection_test.go

```diff
@@ -63,6 +65,113 @@ func (s *testSuite) TestGetPolicySyncMsgFromPoliciesDoesntDowngradeBelowMinimumV
 	s.Equal(policyversion.CurrentVersion().String(), policySync.Policies[0].GetPolicyVersion())
 }
 
+func (s *testSuite) TestSendDeduperStateIfSensorReconciliation() {
+
+	cases := map[string]struct {
+		givenSensorCapabilities       []centralsensor.SensorCapability
+		givenSensorState              central.SensorHello_SensorState
+		assertReconciliationMapClosed bool
+		assertDeduperState            map[string]uint64
+		assertDeduperStateSent        bool
```

Comment:
Nit: Very optional thing - the `assertDeduperState` is omitted in all but 1 case, so how about having it at the end of the struct?
Don´t know why I noticed, but now *that* I have noticed, I cannot unsee it :D
```suggestion
		assertDeduperStateSent        bool
                assertDeduperState            map[string]uint64
```

Reply:
Done